README.md

```diff
@@ -64,18 +63,30 @@ Follow these steps:
     >**NOTE:** For more details about testing, go to the [Testing Kyma](https://kyma-project.io/docs/root/kyma#details-testing-kyma) document.
 
 
-### Project structure
-
-The repository has the following structure:
-
-  ```
-  ├── .github                     # Pull request and issue templates             
-  ├── components                  # Source code of all Kyma components
-  ├── docs                        # Documentation source files
-  ├── installation                # Installation scripts     
-  ├── resources                   # Helm charts and Kubernetes resources for the Kyma installation
-  ├── tests                       # Acceptance tests
-  ├── common                      # Common Go libraries
-  ├── scripts                     # Scripts used for building
-  └── tools                       # Source code of utilities used, for installation, testing, and more
-  ```
+## Kyma users
+
+Kyma is used in these projects:
+
+<ul style="list-style-type: none; display: flex; flex-wrap: wrap;">
+  <li>
+    <a href="https://www.sap.com/index.html" target="_blank">
+      <img src="https://upload.wikimedia.org/wikipedia/commons/5/59/SAP_2011_logo.svg" alt="SAP" style="width: 200px;"/>
+    </a>
+  </li>
+  <li>
+    <a href="https://twiggle.com/" target="_blank">
+      <img src="https://TODO.svg" alt="Twiggle" style="width: 200px;"/>
+    </a>
+  </li>
+  <li>
+    <a href="https://www.mgm-tp.com/en/" target="_blank">
+      <img src="https://TODO.svg" alt="mgm" style="width: 200px;"/>
+    </a>
+  </li>
+  <li>
+    <a href="https://www.netconomy.net/" target="_blank">
+      <img src="https://TODO.svg" alt="NETCONOMY" style="width: 200px;"/>
+    </a>
+</ul>
+
+Read about their [use cases](https://kyma-project.io/#used-by).
```

Comment:
I like we do no duplicate all data from kyma-project website but just put icons and refer to proper view with all details. I would just prefer to also remove the links, so nobody goes to website of the company from the readme

Reply:
Sure 👌